annex/chapter1/adv_examples/include/int_data_producer.h

```diff
@@ -0,0 +1,18 @@
+#include <klepsydra/core/subscriber.h>
```

Comment:
Header file should have an include guard -- #ifndef similar to other header files, based on the name of the header file.

Reply:
Done